jupyter-pytorch-full/rockcraft.yaml

```diff
@@ -141,20 +148,47 @@ parts:
       "${CONDA_BIN}/conda" clean -a -f -y
 
       # Install the jupyter python requirements
+      echo "jupyterlab ==${JUPYTERLAB_VERSION}" >> ${CONDA_DIR}/conda-meta/pinned
+      echo "notebook ==${JUPYTER_VERSION}" >> ${CONDA_DIR}/conda-meta/pinned
+      "${CONDA_BIN}/conda" install -y -q \
+        jupyterlab==${JUPYTERLAB_VERSION} \
+        notebook==${JUPYTER_VERSION}
```

Comment:
Why we don't do clean up like upstream does [here](https://github.com/kubeflow/kubeflow/blob/v1.8.0-rc.6/components/example-notebook-servers/jupyter/Dockerfile#L72)?

Reply:
not needed as we move it to different stage by moving the folder

Reply:
Changed the order